tests/e2e/tests/pilot/testdata/config.yaml.tmpl

```diff
@@ -11,12 +11,12 @@ data:
       drainDuration:          2s
       parentShutdownDuration: 3s
 {{if eq .Auth.String "NONE"}}
-      discoveryAddress:       istio-pilot.{{.IstioNamespace}}:15007
+      discoveryAddress:       istio-pilot.{{.IstioNamespace}}:8080
 {{else}}
       discoveryAddress:       istio-pilot.{{.IstioNamespace}}:15005
 {{end}}
-      discoveryRefreshDelay:  1s
-      connectTimeout:         1s
+      discoveryRefreshDelay:  2s
```

Comment:
Why is this necessary now?

Reply:
I mean this is going to make every test to flake at the first iteration. There is a fixed 3s delay between rule application. You need to increase that delay if we you want to increase refresh delay, or you're just racing them.

Reply:
To test with realistic values - production can't really work with 1 sec refresh.